snow/consensus/snowball/binary_snowflake.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
These headers are on every file in the whole repo. I don't think it makes sense to randomly update these couple.